TestAssets/CrossPublishTestProjects/StandaloneAppCrossPublish/project.json

```diff
@@ -0,0 +1,35 @@
+{
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+  "frameworks": {
+    "netcoreapp1.0": {
+      "imports": [
+        "dnxcore50",
+        "portable-net45+win8"
+      ],
+      "dependencies": {
```

Comment:
I'm not asking to clean up other project.jsons.  I just meant this new one should look more "standard" - like how we generate it in the `dotnet new` template.